services/php-apache-n/Dockerfile

```diff
@@ -3,15 +3,10 @@ FROM tugboatqa/php:${PHP_VERSION}-apache
 ENV COMPOSER_DISCARD_CHANGES=true
 ENV COMPOSER_NO_INTERACTION=true
 ENV COMPOSER_ALLOW_SUPERUSER=true
-ENV INSTALL_NODE_SCRIPT_PATH=/usr/local/bin/install-node-yarn-via-nvm
-RUN curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.0/install.sh | bash - \
-    && . ~/.bashrc \
-    && nvm install --lts \
-    && echo "#!/bin/bash -l" > $INSTALL_NODE_SCRIPT_PATH \
-    && echo "set -e" >> $INSTALL_NODE_SCRIPT_PATH \
-    && echo "nvm install --default" >> $INSTALL_NODE_SCRIPT_PATH \
-    && echo "npm install -g yarn" >> $INSTALL_NODE_SCRIPT_PATH \
-    && chmod +x $INSTALL_NODE_SCRIPT_PATH \
+RUN git clone https://github.com/tj/n.git \
+    && cd n && make install && cd .. \
+    && n lts \
+    && npm install --global yarn \
```

Comment:
This way of installing `n` gives me the heeby jeebies. If it is the official way, so be it, and disregard this comment, but I would much rather we were tying the installation to a tagged release or release bundle rather than whatever happens to be latest master.

Reply:
Good call! Looks like Git’s `clone` command [accepts a `--branch` option](https://git-scm.com/docs/git-clone#Documentation/git-clone.txt---branchltnamegt) which can accept a tag, so something like this _should_ work:
```sh
git clone --branch v8.0.1 https://github.com/tj/n
```

That would be one way to “lock” it, although technically the maintainer could rewrite the tag to point to any commit in the future, but that’s highly unlikely.

I’ll give that a shot and confirm whether it works…

Reply:
Done! I rebuilt `php-apache-n` after that and then rebuilt the Tugboat preview for the related CHQ PR. That Tugboat preview is past the installation of Node v14 now, so it looks like this works as expected.

I’m keeping this as a way to lock our version of n to v8.0.1, with the caveat that the tag is not exactly immutable, but at least we’re protected from any bugs or breaking changes that might get introduced into the project’s HEAD. Thanks for the suggestion, @apotek!

See 4e9911c.

Reply:
The `--branch` method is probably good enough. Another option would be to checkout a specific commit. But this reads more clearly, intent-wise. If you checked out a specific commit, someone would come along later and wonder huh/why? Whereas this documents your intent. Nice stuff.